src/pages/index.js

```diff
@@ -1,54 +1,48 @@
-import './index.css';
-import logo from '../images/Logo.svg';
-// import avatar from '../images/avatar.jpg';
-import pencilIcon from '../images/pencil.svg';
-import plusIcon from '../images/plus.svg';
-
-
-import { enableValidation, settings, resetValidation, disableButton } from "../scripts/validation.js";
-import Api from '../utils/Api.js';
-import { ids } from 'webpack';
+import "./index.css";
+import logo from "../images/Logo.svg";
+import pencilIcon from "../images/pencil.svg";
+import plusIcon from "../images/plus.svg";
+
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+  disableButton,
+} from "../scripts/validation.js";
+import Api from "../utils/Api.js";
+import { setButttonText } from "../utils/helpers.js";
 
 const api = new Api({
   baseUrl: "https://around-api.en.tripleten-services.com/v1",
   headers: {
     authorization: "65069f40-5147-4d93-825a-0d2331637ee5",
-    "Content-Type": "application/json"
-  }
+    "Content-Type": "application/json",
+  },
 });
 
-
-// destructure the second item in the callback of .then()
-  api
-    .getAppInfo()
-    .then(([cards, userData]) => {
-      cards.forEach((item) => {
-        const cardElement = getCardElement(item);
-        cardsList.append(cardElement);
-      });
-
-      // handle the user's information
-      // - set the src of the avatar image
-      // - set the textContent of both the text elements
-      const profileName = document.getElementById("profile__name");
-      const profileAbout = document.getElementById("profile__description");
-      const avatarImg = document.getElementById("avatar");
-
-      profileName.textContent = userData.name;
-      profileAbout.textContent = userData.about;
-      avatarImg.src = userData.avatar;
-    })
-    .catch(console.error);
+api
+  .getAppInfo()
+  .then(([cards, userData]) => {
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.append(cardElement);
+    });
+    const profileName = document.getElementById("profile__name");
+    const profileAbout = document.getElementById("profile__description");
+    const avatarImg = document.getElementById("avatar");
```

Comment:
It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine